docs/tutorials/stateful-application/cassandra.md

```diff
@@ -1,374 +1,134 @@
 ---
-title: "Example: Deploying Cassandra with Stateful Sets"
+title: Deploying Cassandra a StatefulSet
```

Comment:
"Deploying Cassandra a StatefulSet" is really difficult to parse. "Using StatefulSets with Cassandra" perhaps?